docs/requirements/021-API Key管理-实现总结.md

```
@@ -0,0 +1,128 @@
+8. **第七次改**：用户要求"按执行器选择模型" → ApplyProviderRequest 改为 `executor_models: HashMap`
+9. **细节调整**：
+   - 支持 1M 上下文 → 移到模型级
+   - 协议下拉框化
+   - 暗色主题适配
+   - 备份改用 `{原文件名}.bak-{时间戳}` 同目录格式
+   - AtomCode 修复重复 apply
+10. **UI 修复**：点击全选 bug 修复（改用原生 input + 独立状态）
+
+# 7. 测试结果
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)